src/main/java/core/basesyntax/lib/Injector.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.lib;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.exceptions.MissingAnnotationException;
+import core.basesyntax.factory.Factory;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+
+public class Injector {
+    public static Object getInstance(Class<?> clazz) throws NoSuchMethodException,
```

Comment:
<?> parametrizing is equal to <Object> which is by default. So we can leave just Class clazz

Reply:
I know it, but this wildcard annulate warnings from IDEA